README.md

```diff
@@ -51,39 +51,31 @@ idea of the general look and features.
 - File transfers, with previewing of images
 - Audio calls, including group calls
 - Video calls
-- Tox DNS and Tox URI support
-- Translations in 26 languages
+- ToxMe and Tox URI support
+- Translations in 30 languages [](← TODO: add `over` when there'll be more)
 - Avatars
 - Faux offline messages
 - History
 - Screenshots
 - Emoticons
-- Auto-updates on Windows and Mac, packages on Linux
+- Auto-updates on Windows and packages on Linux
 - And many more options!
 
 
-#### Meetings
+#### Organizational stuff
 
-Time & place where qTox committee gathers to discuss. Meetings happen every
-week on Saturday, `16:00 UTC±00:00` on IRC channel [#qtox@freenode].
+Happens in both IRC channel [#qtox@freenode] and on [qTox-dev mailing list].
+If you are interested in participating, **join the channel** and **subscribe to
+the mailing list**.
 
-Logs from the meetings will be posted at least to the mailing list.
+There are available [IRC logs].
```

Comment:
Should be "There are IRC logs available"

Sorry for not using reviewable, doesn't work on mobile...